### PR TITLE
Add cubic phase gate and interferometer, version bump

### PR DESCRIPTION
**Pull request by josh146**
**Description of the Change:**

* Two missing gates have been added. Cubic phase gate has been added
  to the `strawberryfields.fock` device, and the Inteferometer has
  been added to both `strawberryfields.fock` and `strawberryfields.gaussian`.

* Updates the version number and changelog, getting ready for a new release.

**Benefits:** n/a

**Possible Drawbacks:** n/a

**Related GitHub Issues:** n/a

### CODE REVIEW

File: tests/test_gaussian.py [hunk 1]
```diff
@@ -27,6 +27,14 @@
...
 
 
+U = np.array(
+    [
+        [0.83645892 - 0.40533293j, -0.20215326 + 0.30850569j],
+        [-0.23889780 - 0.28101519j, -0.88031770 - 0.29832709j],
+    ]
+)
+
+
```
Comment (antalszava) [inline tests/test_gaussian.py]: It might be nice to have test inputs that are  used in several test files organized into a separate file, as then later it will potentially only need to be changed in one place. Although this might change when transitioning entirely to pytest and right now there are only two tests using them.

File: tests/test_gaussian.py [hunk 2]
```diff
@@ -161,7 +169,9 @@ def test_supported_gaussian_gates(self):
             self.assertTrue(dev.supports_operation(g))
 
             op = getattr(qml.ops, g)
-            if op.num_wires <= 0:
+            if g == "Interferometer":
+                wires = [0, 1]
+            elif op.num_wires <= 0:
```
Comment (antalszava) [inline tests/test_gaussian.py]: Same as in the previous file

File: Makefile [hunk 3]
```diff
@@ -2,7 +2,7 @@ PYTHON3 := $(shell which python3 2>/dev/null)
 
 PYTHON := python3
 COVERAGE := --cov=pennylane_sf --cov-report term-missing --cov-report=html:coverage_html_report
-TESTRUNNER := -m pytest tests
+TESTRUNNER := -m pytest tests --tb=short
```
Comment (antalszava) [inline Makefile]: How come  ``--tb=short`` was required?

Comment (josh146) [reply Makefile]: By default pytest tracebacks are very long and hard to parse easily. This just shortens it, and will make them easier to read when on Travis.

File: tests/test_fock.py [hunk 4]
```diff
@@ -172,14 +180,17 @@ def test_supported_fock_gates(self):
...
+                wires = [0, 1]
+            elif op.num_wires <= 0:
                 wires = [0]
             else:
                 wires = list(range(op.num_wires))
 
             @qml.qnode(dev)
             def circuit(*args):
                 qml.TwoModeSqueezing(0.1, 0, wires=[0, 1])
+                print(args, wires)
```
[the remark this answers is not in the document]
Comment (josh146) [reply tests/test_fock.py]: Good catch!

File: tests/test_fock.py [hunk 5]
```diff
@@ -172,14 +180,17 @@ def test_supported_fock_gates(self):
             self.assertTrue(dev.supports_operation(g))
 
             op = getattr(qml.ops, g)
-            if op.num_wires <= 0:
+            if g == "Interferometer":
+                wires = [0, 1]
+            elif op.num_wires <= 0:
```
[the remark this answers is not in the document]
Comment (josh146) [reply tests/test_fock.py]: Changed!

File: tests/test_gaussian.py (hunk 2 above)
Comment (josh146) [reply tests/test_gaussian.py]: Changed!

File: tests/test_gaussian.py (hunk 1 above)
Comment (josh146) [reply tests/test_gaussian.py]: Yep, this is something we should look at in a future PR (port to pytest, and clean up existing tests)
